api/src/pcapi/core/offerers/factories.py

```diff
@@ -437,3 +437,26 @@ class Meta:
         "transport_modality": [acceslibre_enum.PARKING_NEARBY],
     }
     lastUpdateAtProvider = datetime.datetime(2024, 3, 1, 0, 0)
+
+
+class AddressFactory(BaseFactory):
+    street = "1 boulevard Poissonnière"
+    postalCode = "75002"
+    city = "Paris"
+    country = "France"
+    latitude: float | None = 48.87004
+    longitude: float | None = 2.37850
```

Comment:
Détail : ces coordonnées GPS sont dans le 11ème.
Pour la correspondance avec la réalité souhaitée par François : 
```suggestion
    latitude: float | None = 48.87052
    longitude: float | None = 2.34761
```

Reply:
J'ai mis des valeurs un peu plus précise